src/libraries/Microsoft.Win32.SystemEvents/ref/Microsoft.Win32.SystemEvents.cs

```diff
@@ -59,6 +59,7 @@ public sealed partial class SystemEvents
         internal SystemEvents() { }
         public static event System.EventHandler? DisplaySettingsChanged { add { } remove { } }
         public static event System.EventHandler? DisplaySettingsChanging { add { } remove { } }
+        [System.Obsolete("SystemEvents.EventsThreadShutdown callbacks are not run before process exits. Use AppDomain.ProcessExit instead.")]
```

Comment:
Don't we need to pass UrlFormat and DiagnosticId in here as well?

Reply:
The ref project targets netstandard2.0 and those don't exist there.

Reply:
The ref project targets many TFMs: https://github.com/dotnet/runtime/blob/a3fe47ef1a8def24e8d64c305172199ae5a4ed07/src/libraries/Microsoft.Win32.SystemEvents/ref/Microsoft.Win32.SystemEvents.csproj#L3

We usually polyfill support for that API by setting the following property: `<IncludeInternalObsoleteAttribute>true</IncludeInternalObsoleteAttribute>`: https://github.com/dotnet/runtime/blob/a3fe47ef1a8def24e8d64c305172199ae5a4ed07/src/libraries/Directory.Build.targets#L194C25-L194C57

Reply:
So all of my other comments will be resolved by setting this and using the newer API in both ref and src. Then you can delete the changes to the CompatibilitySuppressions.xml file and the manual include of the Obsoletions.cs file.

Reply:
It looks like we need to keep the manual include of Obsoletions.cs based on https://dev.azure.com/dnceng-public/public/_build/results?buildId=852651&view=results

Reply:
Hm, the build looks to be generating a file called SystemEvents.notsupported.cs in artifacts folder and it cannot find Obsoletions.cs. I see [LowMemory](https://github.com/dotnet/runtime/blob/7ee2524959db2f4c8cb9a735f882f4264e49a3e5/src/libraries/Microsoft.Win32.SystemEvents/ref/Microsoft.Win32.SystemEvents.cs#L67), which is another obsolete API has hardcoded obsoletion message. Should the obsoletion message, UrlFormat, and DiagnosticId for EventsThreadShutdown be hardcoded as well?